documentation/system-design/spec/spec.ts

```diff
@@ -315,15 +316,15 @@ const PostServiceModule: Module = {
   associatedRequirements: [FR.FR1, FR.FR2, FR.FR3],
   contents: {
     syntax: {
-      exportedTypes: [PostWithUserType, CreateablePostType, UpdatablePostType],
+      exportedTypes: [PostWithUserType, CreatablePostType, UpdatablePostType],
       exportedAccessPrograms: [
         {
           name: "getPost",
           in: {
             id: "string",
             includeUser: "boolean",
           },
-          out: [PostType, PostWithUserType],
+          out: [PostType, PostWithUserType, Null],
```

Comment:
Lowercase "null" as a string is used elsewhere in the spec as a type. This can be just a string or you can update the rest of the cases to use this new type.

Reply:
switched over to using just a string